### ISSUE TITLE
Welcome / AA screen dissapear after clicking on AA icon.

### ISSUE THREAD

**Issue by egisz** [openDsh/dash#89]
# Issue

Welcome / AA screen dissapear after clicking on AA icon.

## Hardware

| Device             | Connection Method | Screen                       | Mobile Device     | Android Auto Version |
| ------------------ | ----------------- | ---------------------------- | ----------------- | -------------------- |
| Raspberry Pi 4 4GB | USB Cable         | irrelevant | Samsung Galaxy S21 | v.1.EXAMPLE          |

Installation Method: Install.sh 

## Description of problem:

Welcome / AA screen dissapear after clicking on AA icon.
![20210806_092213](https://user-images.githubusercontent.com/1941760/128470674-03a0f940-a2bc-424e-a13d-9436e4275562.jpg)

## Expected Result:

In case device is not connected, **Connect device to start Android Auto** screen should be visible in AA tab.
Plus AA settings icon not visible either

## Problem-relevant steps to reproduce:

1. Open Dash
1. Click on AA icon, screen becomes empty/blank. No way to get into AA settings until Dash restart

**Comment by rsjudka** [comment]
hmm.. ill have to look into this, havent been on the develop branch in a while, but yeah the connect screen should always be visible if AA isnt connected

out of curiosity.. what is shown then on the page?

**Comment by egisz** [comment]
updated OP, added screenshot.

**Comment by icecube45** [comment]
potential side effect of #84 perhaps?

**Comment by egisz** [comment]
> potential side effect of #84 perhaps?

Yes... pushed potential fix to aa_fullscreen  branch (#87). Looks OK on PC, will test it on Raspberry.
